hardhat/hardhat.multiplewinners.js

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
Are these mainnet fork tests? Where are they used?
Maybe it should be placed in the `scripts/fork` folder.

Reply:
These are not mainnet forks. Simply unit tests to check coverage.